From c5f80115a04b3714b73e1f4dc1359b8e51ae8aa0 Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Sun, 26 Feb 2017 17:12:56 -0800
Subject: [PATCH] Remove getpass definition

It is a legacy function and musl already provides an implementation.
---
 lib/libc/gen/readpassphrase.c | 12 ++----------
 1 file changed, 2 insertions(+), 10 deletions(-)

diff --git a/lib/libc/gen/readpassphrase.c b/lib/libc/gen/readpassphrase.c
index 01bff7d306c..f9d91263bc8 100644
--- a/lib/libc/gen/readpassphrase.c
+++ b/lib/libc/gen/readpassphrase.c
@@ -198,16 +198,8 @@ restart:
 }
 DEF_WEAK(readpassphrase);
 
-char *
-getpass(const char *prompt)
-{
-	static char buf[_PASSWORD_LEN + 1];
-
-	return(readpassphrase(prompt, buf, sizeof(buf), RPP_ECHO_OFF));
-}
-
-static void handler(int s)
+static void
+handler(int s)
 {
-
 	signo[s] = 1;
 }
-- 
2.54.0

